src/App.js

```diff
@@ -1,28 +1,28 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react'
+import io from 'socket.io-client'
+import OAuth from './OAuth'
+import './App.css'
 
-class App extends Component {
+const API_URL = 'http://localhost:8080'; // TODO change for production
```

Comment:
can we do an env for this? we have our prod url: `http://www.photoswipeapp.com/`

for the time being we only have prod, but i think at some point i can figure out how to add a subdomain for staging.